main.py

```diff
@@ -69,5 +69,16 @@ async def on_message_edit(old: discord.Message, new: discord.Message):
     await pgbot.message_edit(old, new)
 
 
+@bot.event
+async def on_raw_reaction_add(payload):
+    """
+    This function is called for every reaction added by user.
+    """
+    if payload.member.bot or payload.member is None:
```

Comment:
Still gonna error SMH

Reply:
`if payload.member is None or payload.member.bot:`

Reply:
oof